exercises/space-age/canonical-data.json

```diff
@@ -80,6 +80,16 @@
         "seconds": 1821023456
       },
       "expected": 0.35
+    },
+    {
+      "uuid": "57b96e2a-1178-40b7-b34d-f3c9c34e4bf4",
+      "description": "age on Pluto",
+      "property": "age",
+      "input": {
+        "planet": "Pluto",
+        "seconds": 680804807
+      },
+      "expected": 0
```

Comment:
I can't agree with the answer being 0. Your age isn't 0 on Pluto. Instead it should be `{"error": "not a planet"}`, or something similar.